src/screens/MainApp/Profile/styles.js

```diff
@@ -1,46 +1,44 @@
 import { StyleSheet } from 'react-native'
-import { moderateScale } from 'react-native-size-matters'
+import { scale, verticalScale } from 'react-native-size-matters/extend'
 import { theme } from '~/constants'
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    flexDirection: 'column'
+    flexDirection: 'column',
+    padding: theme.sizes.margin * 2
   },
   title: {
-    textAlign: 'center',
-    padding: moderateScale(10)
+    fontSize: 24,
```

Comment:
Please consider using scaling units for `fontSizes` too or prefer the theme one. If it seems small compared to the design then we change that value in the theme.

Reply:
I'll change it to use scaling units. The theme doesn't contain that size.

Reply:
@jmbidev don't all titles have to be the same size? Why don't we add this to the theme?
cc @MPucheta

Reply:
I only needed this size in this screen. The other titles are defined as h1 h2 h3 y h4 on theme with sizes 20, 18, 16, 14 respectively.

Reply:
Titles do have the same size. Which is the the one applied in the `Title` component using `h1` from the theme (which has a size of `moderateScale(20,1)`. In the case that value seems small compared to the designs, we can change it in only one place (although the current one seems fine by me, it's the one used in the Onboarding, Auth, MyDiagnoses, etc).

Reply:
@jmbidev What I'm asking is _why_ does this screen need a different title size than the other screens that Mauro enumerated above. All titles have to be the same size on all screens. If they're different than the design, then we have to change all titles, not just this one.

Reply:
Because the size used on the screen design was different from the sizes defined in theme. In the design Agos used size 24 and the defined sizes are the ones I enumerated above. Also in the styleguide that Agos created there was no size 24 defined, I think that could be a reason because it's not defined on theme. 

@nahudalla

Reply:
There is not big trouble I can remove size property and modify h1 to be size 24

Reply:
The problem here was that I already used Title element on those screens with font 20 titles (this happens because when I wrote the screens I used the element that fit better the size property).  But I suppose those ones are Subtitles instead.

Reply:
@nahudalla  @MPucheta